napari/layers/labels/labels.py

```diff
@@ -341,6 +344,20 @@ def mode(self, mode: Union[str, Mode]):
         self.events.mode(mode=mode)
         self.refresh()
 
+    @property
+    def overwrite(self):
+        """
+        Defines if painting should overwrite existing labels, default to true to allow overwrite.
```

Comment:
I'll do this for @jni :)
please make this (and other) docstrings [PEP-257](https://www.python.org/dev/peps/pep-0257/) compliant.  That is: one line, followed by period.  double space, then extra info:
```python
def function(arg: Type):
    """One-liner not exceeding line-length, ending in period.

    Any additional info you'd like to provide...
    """
```